### DISCUSSION TITLE
Want have a conbination of `Neotree`'s `git_status` and `diffview.nvim`'s diff view

### DISCUSSION THREAD

**Discussion by LintaoAmons** [nvim-neo-tree/neo-tree.nvim#1072] [General]
Something like this~ 🥲 

![image](https://github.com/nvim-neo-tree/neo-tree.nvim/assets/95092244/40f71b1f-87ae-406d-812d-a7a3896ea3e3)

**Comment by miversen33** [comment]
I've been toying with the idea of making a new git source, something more akin to what the version control source in vscode is like. I love this idea :)

**Reply by Azarattum** [reply]
Any luck so far?
